### DISCUSSION TITLE
Detect developer option if enabled in the app

### DISCUSSION THREAD

**Discussion by chie-ai** [quasarframework/quasar#10422] [CLI - Capacitor / Cordova modes]
Is there a way to detect developer option in the app?

I just want to detect developer option if enabled to let not the user continue using a certain mode in the app.

Please help I need to solve this.

**Comment by ontwikkelfabriek** [comment]
@chie-ai 

> developer option

What do you mean this exactly?

**Comment by pikil** [comment]
you can use ```process.env.DEBUGGING``` which is a built in property for your Quasar-based app.

You can also add DEBUG_MODE: true/false to bulid->env parameters for your app if there is some certain case is needed.
```
build: {
      env: ctx.dev
        ? {
          DEBUG_MODE: true
        } : {
          DEBUG_MODE: false
        }
...
```
